apps/api/src/server/repositories/case.repository.js

```diff
@@ -58,6 +58,9 @@ export const getByStatus = (statusArray) => {
 	return databaseConnector.case.findMany({
 		orderBy: [{ ApplicationDetails: { subSector: { abbreviation: 'asc' } } }],
 		where: {
+			NOT: {
```

Comment:
this seems to ignore draft cases (where reference is NULL) - and so draft cases are not being returned in the search anymore.  See bug: https://pins-ds.atlassian.net/browse/APPLICS-61

Reply:
interesting, fix incoming!